src/Stratis.Bitcoin.IntegrationTests/RPC/RPCTestsMutable.cs

```diff
@@ -21,11 +21,18 @@ public void TestRpcGetBalanceIsSuccessful()
                 builder.StartAll();
                 RPCClient rpcClient = node.CreateRPCClient();
 
-                TestHelper.MineBlocks(node, 2);
-                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == 2);
+                int maturity = (int)KnownNetworks.RegTest.Consensus.CoinbaseMaturity;
+
+                // This shouldn't be necessary but is required to make this test pass.
+                // - looks like a bug in the wallet?
+                maturity--; 
+                
+                TestHelper.MineBlocks(node, maturity);
+                TestHelper.WaitLoop(() => node.FullNode.GetBlockStoreTip().Height == maturity);                             
+                Assert.Equal(Money.Zero, rpcClient.GetBalance());
 
-                Money balance = rpcClient.GetBalance();
-                Assert.Equal(Money.Coins(100), balance);
+                rpcClient.Generate(1);
```

Comment:
Why not use MineBlocks?
```suggestion
                TestHelper.MineBlocks(node, 1);
```